tests/lambda_handlers_tests/test_dashboardHandler.py

```diff
@@ -1,27 +1,357 @@
 from __future__ import absolute_import, unicode_literals, print_function
```

Comment:
This whole test file is JUST to make sure the handler has  been coded properly, without executing the TxManager object code. That's why the TxManager call is mocked in every single one of these. At most this should just add testing that the max failures parameter is gotten properly from the request. 

The testing of  the actual code of TxManager should be in the manager test directory, treated independently of the request handler.